tests/test_models.py

```diff
@@ -93,21 +102,34 @@ def test_empty_social_links(self):
         assert_equal(self.user.social_links, {})
         assert_equal(len(self.user.social_links), 0)
 
-    def test_personal_site_unchanged(self):
-        self.user.social = {'personal': 'http://cos.io/'}
+    def test_profile_website_site_unchanged(self):
+        self.user.social = {'profileWebsites': ['http://cos.io/']}
```

Comment:
Minor change, but could you change this test name to `test_profile_website_unchanged`? `website_site` sounds redundant.